### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by aledbf**
## Description

- Update dependencies
- Update go to v1.19
- Update buildkit to v0.10.4
- Remove estargz from build
- gopkg.in/yaml to v3

### CODE REVIEW

File: pkg/dazzle/project.go [hunk 1]
```diff
@@ -330,7 +331,7 @@ func loadChunks(dir fs.FS, contextBase, base, name string) (res []ProjectChunk,
 		} else if err != nil {
 			return nil, fmt.Errorf("%s: cannot read tests.yaml: %w", dir, err)
 		}
-		err = yaml.UnmarshalStrict(tf, &chk.Tests)
+		err = yaml.Unmarshal(tf, &chk.Tests)
```
Comment (kylos101) [inline pkg/dazzle/project.go:334]: To preserve [strict mode behavior](https://ubuntu.com/blog/api-v3-of-the-yaml-package-for-go-is-available#:~:text=Strict%20mode%20is%20now%20Decoder.KnownFields), should we do something like this on L345?

```go
decoder := yaml.NewDecoder(fd)
decoder.KnownFields(true)
err = decoder.Decode(&cfg)
```

**Review by kylos101 — COMMENTED** [review]
:wave: @aledbf just one question, everything else is :+1: :+1:

File: pkg/dazzle/project.go (hunk 1 above)
Comment (aledbf) [reply pkg/dazzle/project.go]: Good point. Fixed.

**Review by kylos101 — APPROVED** [review]
:train: choo choo!

**Pull request merged**